src/components/Vuetify/mixins/goTo.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Modified from https://github.com/alamcordeiro/vue-smooth-scroll
+ */
+
+function easeInOutCubic (t) { return t < .5 ? 4 * t * t * t : (t - 1) * (2 * t - 2) * (2 * t - 2) + 1 }
+
+export default function (target) {
+  if (typeof window === 'undefined') return
+
+  let duration = 500
+  let start = Date.now()
```

Comment:
`Date.now()` is not suitable for animation, use `performance.now()` instead.

https://developers.google.com/web/tools/lighthouse/audits/date-now